py/tests/test_subscriptions.py

```diff
@@ -1,20 +1,118 @@
+from flask import json
 from tests.test_case import *
-from app.pcasts.dao import subscriptions_dao, series_dao
+from app import constants
+from app.pcasts.dao import subscriptions_dao, series_dao, users_dao
 from app.pcasts.controllers import series_subscriptions_controller
 
 class SubscriptionsTestCase(TestCase):
 
   def setUp(self):
     super(SubscriptionsTestCase, self).setUp()
-    Subscription.query.delete() # delete all existing subscriptions
+    Subscription.query.delete()
     series_dao.clear_all_subscriber_counts()
     db_session_commit()
 
+  def test_create_subscription(self):
+    series_id1 = '1211520413'
+    series_id2 = '1210383304'
+    series = Series.query.filter(Series.id == series_id1).first()
+    self.assertEquals(series.subscribers_count, 0)
+
+    self.app.post('api/v1/subscriptions/{}/'.format(series_id1))
+    series = Series.query.filter(Series.id == series_id1).first()
+    self.assertEquals(series.subscribers_count, 1)
+
+    with self.assertRaises(Exception):
+      self.app.post('api/v1/subscriptions/{}/'.format(series_id1))
+
+    self.app.post('api/v1/subscriptions/{}/'.format(series_id2))
+    series = Series.query.filter(Series.id == series_id1).first()
+    self.assertEquals(series.subscribers_count, 1)
+
   def test_get_subscriptions(self):
-    assert not subscriptions_dao.get_user_subscriptions(1)
+    series_id1 = '1211520413'
+    series_id2 = '1210383304'
+    self.app.post('api/v1/subscriptions/{}/'.format(series_id1))
+    self.app.post('api/v1/subscriptions/{}/'.format(series_id2))
 
-  def test_create_subscription(self):
-    response = self.app.post('api/v1/subscriptions/1211520413/')
+    response = self.app.get('api/v1/subscriptions/{}/?offset={}&max={}'.\
+        format(series_id1, 0, 1))
+    data = json.loads(response.data)
+    self.assertEquals(len(data['data']['subscriptions']), 1)
+    self.assertEquals(
+        data['data']['subscriptions'][0]['series_id'], int(series_id1)
+    )
+    self.assertEquals(
+        data['data']['subscriptions'][0]['user']['google_id'],
+        constants.TEST_USER_GOOGLE_ID1
+    )
+
+    response = self.app.get('api/v1/subscriptions/{}/?offset={}&max={}'.\
```

Comment:
you don't need the `\` at the end of this line because you're in parentheses, the next line can just be `.format(series_id1, 0, 0))`

Reply:
will change in my next PR